crates/bevy_ecs/src/schedule/condition.rs

```diff
@@ -877,25 +874,22 @@ pub mod common_conditions {
     /// app.run(&mut world);
     /// assert_eq!(world.contains_resource::<MyResource>(), true);
     /// ```
-    pub fn resource_changed_or_removed<T>() -> impl FnMut(Option<Res<T>>) -> bool + Clone
```

Comment:
Does this work right? I'm worried it's cloning something else, since Systems aren't always Clone.

Reply:
A function pointer is always clonable, so I think it should work the same as it would for any function system.